GitCommands/Git/GitModule.cs

```diff
@@ -2467,9 +2473,9 @@ public IReadOnlyList<GitItemStatus> GetStashDiffFiles(string stashName)
             // shows untracked files
             var args = new GitArgumentBuilder("log")
             {
-                $"{stashName}^3",
                 "--pretty=format:\"%T\"",
-                "--max-count=1"
+                "--max-count=1",
+                $"-- {stashName}^3"
```

Comment:
-- separates paths, stashNane is a ref
The order is correct now though

Reply:
The commit message contains the git error:

```
Fixup git arguments for GetStashDiffFiles

fatal: ambiguous argument 'stash@{0}^3': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions
```

sounds a bit confusing to me; but works well

Reply:
stashName is not a path, so it should not be separated with `--`

Reply:
Without "--", git outputs the above _fatal_ error with non-zero exit code.